back/src/main/resources/properties/resilience.yml

```
@@ -0,0 +1,58 @@
+          - org.springframework.dao.TransientDataAccessException
+          - java.sql.SQLTransientException
+          - java.sql.SQLRecoverableException
+        ignoreExceptions:
+          - java.lang.IllegalArgumentException # 재시도해도 소용없는 것들
+  retry:
+    instances:
+      dbWriter:
+        maxAttempts: 1
+        waitDuration: 0
+        retryExceptions:
+          - org.springframework.dao.TransientDataAccessException
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)